lib/resources/r4/related_person_patch.yaml

```diff
@@ -92,6 +92,33 @@ operations:
         <li>If `line` is provided, only the first four values are used. Additional values will be ignored.</li>
       </ul>
 
+  - name: test-address-id
+    path: /address/{index}/id
+    operation: test
+    type: string
+    url: https://hl7.org/fhir/R4/element-definitions.html#Element.id
+    description: Test the value of the `Address.id` of the address at the given <code>{index}</code> in the list of related person addresses.
+    example: |
+      {
+        "op": "test",
+        "path": "/address/0/id",
+        "value": "CI-24079465-0"
+      }
+  - name: remove-address
```

Comment:
nit: add a space between the two '-name' blocks

Reply:
https://github.com/cerner/fhir.cerner.com/commit/9881b1112521bc3451461f10b0eb845582d0952b#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR107